src/avfoundation.cs

```diff
@@ -8716,15 +8718,16 @@ interface AVCaptureDepthDataOutputDelegate
 	interface AVCaptureDepthDataOutput
 	{
 		[Export ("setDelegate:callbackQueue:")]
-		void SetDelegate (IAVCaptureDepthDataOutputDelegate del, DispatchQueue callbackQueue);
+		void SetDelegate ([NullAllowed] IAVCaptureDepthDataOutputDelegate del,[NullAllowed] DispatchQueue callbackQueue);
 
 		[Wrap ("WeakDelegate")]
+		[NullAllowed]
```

Comment:
Assumed any "wrap" delegate should have the same nullability attributes as the `WeakDelegate` - not sure I've captured all occurrences :-?

Reply:
yes, the inside of the `[Wrap]` is text so the generator can't make all the checks that it can do normally